manifests/init.pp

```diff
@@ -140,6 +140,7 @@
     enable_puppet          => true,
     enable_docker          => true,
     enable_parent_node     => true,
+    ca_cert                => $::certs::ca_cert,
```

Comment:
This doesn't need tobe set -- see https://github.com/Katello/puppet-katello/pull/102